weblate/trans/tests/test_edit.py

```diff
@@ -817,22 +819,35 @@ def test_edit_check(self):
             reverse("js-ignore-check", kwargs={"check_id": check_id})
         )
         self.assertContains(response, "ok")
+
         # Should have one less failing check
         unit = self.get_unit()
         self.assertFalse(unit.has_failing_check)
         self.assertEqual(len(unit.all_checks), 1)
         self.assertEqual(len(unit.active_checks), 0)
         self.assertEqual(unit.translation.stats.allchecks, 0)
+
         # Ignore check for all languages
         ignore_url = reverse("js-ignore-check-source", kwargs={"check_id": check_id})
         response = self.client.post(ignore_url)
         self.assertEqual(response.status_code, 403)
         self.user.is_superuser = True
         self.user.save()
         response = self.client.post(ignore_url)
-        self.assertContains(response, "ok")
+        self.assertEqual(response.headers["Content-Type"], "application/json")
+
         # Should have one less check
         unit = self.get_unit()
+        obj = get_object_or_404(Check, pk=int(check_id))
```

Comment:
No need to go via get_object_or_404:

```suggestion
        obj = Check.objects.get(pk=int(check_id))
```